imap_processing/swapi/l1/swapi_l1.py

```diff
@@ -485,12 +486,19 @@ def process_swapi_science(
     # ===================================================================
     # Quality flags
     # ===================================================================
-    quality_flags_data = np.zeros((total_full_sweeps, 72), np.uint16)
+    quality_flags_data = np.zeros((total_full_sweeps, 72), dtype=np.uint16)
 
     # Add science data quality flags
-    quality_flags_data[pcem_compression_flags == 1] |= SWAPIFlags.SWP_PCEM_COMP.value
-    quality_flags_data[scem_compression_flags == 1] |= SWAPIFlags.SWP_SCEM_COMP.value
-    quality_flags_data[coin_compression_flags == 1] |= SWAPIFlags.SWP_COIN_COMP.value
+    # Have to match datatype to bitwise OR
+    quality_flags_data[pcem_compression_flags == 1] |= np.uint16(
+        SWAPIFlags.SWP_PCEM_COMP
+    )
+    quality_flags_data[scem_compression_flags == 1] |= np.uint16(
+        SWAPIFlags.SWP_SCEM_COMP
+    )
+    quality_flags_data[coin_compression_flags == 1] |= np.uint16(
+        SWAPIFlags.SWP_COIN_COMP
+    )
```

Comment:
There was casting error that only happens when code is running in batch job. This was fixing that.